apps/PlayniteWebPlugin/src/PlayniteWeb.cs

```diff
@@ -27,323 +27,371 @@ namespace PlayniteWeb
 
       publisher.StartDisconnect().Wait();
     }
 
     public override void OnApplicationStarted(OnApplicationStartedEventArgs args)
-    {
-      settings.OnVerifySettings += HandleVerifySettings;
-      try
-      {
-        StartConnection(settings.Settings);
-      }
-      catch (Exception ex) { }
-
-      subscriber.OnLibraryRequest += Publisher_LibraryRefreshRequest;
-      subscriber.OnStartGameRequest += Subscriber_OnStartGameRequest;
-      subscriber.OnInstallGameRequest += Subscriber_OnInstallGameRequest;
-      subscriber.OnUninstallGameRequest += Subscriber_OnUninstallGameRequest;
-
-
-      gameUpdates.Subscribe(e => HandleGameUpdated(this, e));
-      platformUpdates.Subscribe(e => HandlePlatformUpdated(this, e));
-      otherEntityUpdates.Subscribe(e => HandleOtherGameEntitiesUpdated(this, e));
-      collectionUpdates.Subscribe(e => HandleCollectionUpdate(this, e));
-    }
-
-    private void Subscriber_OnUninstallGameRequest(object sender, Guid e)
-    {
-      var game = PlayniteApi.Database.Games.First(g => g.Id == e);
-      if (game.IsInstalled)
-      {
-        PlayniteApi.UninstallGame(e);
-        var gameStatePublisher = new PublishGameState((IMqttClient)publisher, topicManager, serializer);
-        Task.WaitAll(gameStatePublisher.Publish(game).ToArray());
-      }
-    }
-
-    private void Subscriber_OnInstallGameRequest(object sender, Guid e)
-    {
-      var game = PlayniteApi.Database.Games.First(g => g.Id == e);
-      if (!game.IsInstalled)
-      {
-        PlayniteApi.InstallGame(e);
-        var gameStatePublisher = new PublishGameState((IMqttClient)publisher, topicManager, serializer);
-        Task.WaitAll(gameStatePublisher.Publish(game).ToArray());
-      }
-    }
-
-    private void HandleVerifySettings(object sender, PlayniteWebSettings e)
-    {
-      StartConnection(e);
-    }
-
-    private void StartConnection(PlayniteWebSettings settings)
-    {
+    {
+      settings.OnVerifySettings += HandleVerifySettings;
+      try
+      {
+        StartConnection(settings.Settings);
+      }
+      catch (Exception ex) { }
+
+      subscriber.OnLibraryRequest += Publisher_LibraryRefreshRequest;
+      subscriber.OnStartGameRequest += Subscriber_OnStartGameRequest;
+      subscriber.OnInstallGameRequest += Subscriber_OnInstallGameRequest;
+      subscriber.OnUninstallGameRequest += Subscriber_OnUninstallGameRequest;
+
+
+      gameUpdates.Subscribe(e => HandleGameUpdated(this, e));
+      platformUpdates.Subscribe(e => HandlePlatformUpdated(this, e));
+      otherEntityUpdates.Subscribe(e => HandleOtherGameEntitiesUpdated(this, e));
+      collectionUpdates.Subscribe(e => HandleCollectionUpdate(this, e));
+    }
+
+    private void Subscriber_OnUninstallGameRequest(object sender, Guid e)
+    {
+      try
+      {
+        var game = PlayniteApi.Database.Games.FirstOrDefault(g => g.Id == e);
+        if (game == null)
+        {
+          Logger.Debug($"Game with ID {e} not found for uninstall.");

```

Comment:
Logger methods are not static. To correct this, we'll need to instantiate a a private logger class member and use it in place of `Logger.*`.

I'll add a separate comment for the logger class member addition.

```suggestion
          logger.Debug($"Game with ID {e} not found for uninstall.");

```